.gitignore

```diff
@@ -3,3 +3,4 @@
 .mypy_cache/
 .pytest_cache/
 __pycache__/
+*.pyc
```

Comment:
@acroz I add this. I think you were not seeing those pyc's because you're ignoring them in your global git config?

Reply:
Well I think it's just that I didn't run the code in a way that made .pyc files in the main source directory. I don't have it in my global git ignores. It's useful to include though, I'll add it.

Reply:
I agree with @acroz on not pushing without coordination.